translator/test/src/test/resources/features/TranslatorUnitTests.feature

```diff
@@ -0,0 +1,217 @@
+###############################################################################
+# Copyright (c) 2020 Eurotech and/or its affiliates and others
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     Eurotech
+###############################################################################
+@unit
+@translator
+
+Feature: Translator Service
+  The Translator Service is responsible for messaging operations between Kura and Kapua.
+
+  #KapuaTranslatorApi
+
+  Scenario: Finding translator "TranslatorAppCommandKapuaKura"
+  Trying to make translation from CommandRequestMessage to KuraRequestMessage.
+
+    When I try to translate from "org.eclipse.kapua.service.device.management.command.message.internal.CommandRequestMessage" to "org.eclipse.kapua.service.device.call.message.kura.app.request.KuraRequestMessage"
+    Then Translator "TranslatorAppCommandKapuaKura" is found
+    And No exception was thrown
+
+  Scenario: Finding null translator
+  Trying to do translation without messages. NPE exception should be thrown.
+
+    Given I expect the exception "NullPointerException" with the text "*"
+    And I try to translate from "" to ""
+    Then An exception was thrown
+
+  Scenario: Finding wrong translator
+  Trying to make translation from AssetRequestMessage to AssetResponseMessage.
+  Translator doesn't exist.
+
+    Given I expect the exception "KapuaRuntimeException" with the text "*"
+    And I try to translate from "org.eclipse.kapua.service.device.management.asset.message.internal.AssetRequestMessage" to "org.eclipse.kapua.service.device.management.asset.message.internal.AssetResponseMessage"
+    Then An exception was thrown
+
+  #TranslatorDataMqttKura
+
+  Scenario: Translating of mqtt message with invalid payload and valid topic into kura data message
+  Creating mqtt message with invalid payload and valid topic. Trying to translate it into kura data message.
+  Checking if kura data message with valid channel and byte[] payload is received.
+
+    Given I create mqtt message with invalid payload "invalidPayload" and valid topic "kapua-sys/rpione3/DEPLOY-V2/GET/packages"
```

Comment:
I think here should be "validPayload", since tests is succcessfull and it does not return any exceptions. This is just for the sake of better understanding of the test itself.